lib/Models/WebMapServiceCatalogItem.js

```diff
@@ -567,6 +571,11 @@ WebMapServiceCatalogItem.prototype._load = function() {
     var that = this;
     var promises = [];
 
+    if (this._lazyLoadInterval) {
+        this._lazyLoadInterval();
+        delete this._lazyLoadInterval;
```

Comment:
We usually prefer to set things to `undefined` instead of using `delete` because changing the shape of the object causes chrome performance problems.